securedrop_client/logic.py

```diff
@@ -412,8 +421,17 @@ def on_sync_success(self) -> None:
 
     def on_sync_failure(self, result: Exception) -> None:
         """
-        Called when syncronisation of data via the API queue fails.
+        Called when syncronisation of data via the API fails. Just log and try again since this is
+        a background task.
+        """
+        logger.debug('The SecureDrop server cannot be reached due to Error: {}'.format(result))
+        self.sync_api()
```

Comment:
recapping from synchronous chat just now: we are going to remove the self.sync_api() call here and instead resume the queues. By doing so, we are also closing #491, and we are going to treat the `MetadataSyncJob` as our network health checker. The queue pause / retry error message will continue to appear as per the acceptance criteria in that ticket until the network action is confirmed to succeed. The removal of that error message will be doing in the `on_sync_success` callback.